### DISCUSSION TITLE
Multiple configurations switchable with USR button or similar technique

### DISCUSSION THREAD

**Discussion by r1me** [frntc/SIDKick-pico#18] [Ideas]
A feature request to support multiple (three?) configurations. i.e. switchable between 2x 8580 and 2x 6581 without need to access configuration tool. One of the Pico lines/USR button could be used. A physical switch could be routed outside of the case or keyman64.

**Comment by frntc** [comment]
Hi,

the large SIDKick (based on the Teensy 4.1) had this feature and my impression was that only very few people used it (me included). I might integrate this at some point, but there's not schedule for this yet.
